web-client/Cargo.toml

```
@@ -45,3 +45,18 @@ features = [
     "wasm-websocket",
     "web-logging",
 ]
+
+[dev-dependencies]
+parking_lot = "0.12.1"
+wasm-bindgen-test = "0.3.33"
+beserial = { path = "../beserial", features = ["derive"] }
+
+nimiq-bls = {path = "../bls"}
```

`beserial` is an internal crate. I'd suggest:
```suggestion
wasm-bindgen-test = "0.3.33"

beserial = { path = "../beserial", features = ["derive"] }
nimiq-bls = {path = "../bls"}
```